scripts/publish.sh

```diff
@@ -74,24 +75,11 @@ for CRATE_DIR in ${ORDER[@]}; do
 		fi
 	done
 
-	# Wait again to make sure that the new version is published and available.
```

Comment:
Latest `cargo publish`does indeed wait for registry to available so this is not needed anymore.